File: Python3/58.Length_Of_Last_Word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        new_str = s.strip()

        length = 0
        n = len(new_str)
        for i in range(n):
            if new_str[n - i - 1] == ' ':
                break
            else:
                length += 1
        return length

File: Java/58.Length_Of_Last_Word.java
class Solution {
    public int lengthOfLastWord(String s) {
        String new_str = s.trim();

        int length = 0;
        for (int i = new_str.length() - 1; i >= 0; --i) {
            if (new_str.charAt(i) == ' ') break;
            else length += 1;
        }   
        return length;
    }
}}